brambling/models.py

```diff
@@ -651,13 +664,13 @@ def remove_from_cart(self, bought_item):
             self.cart_start_time = None
             self.save()
 
-    def mark_cart_paid(self):
+    def mark_cart_paid(self, status):
```

Comment:
Looking at this again, I think mark_cart_paid is appropriate. The "cart" - i.e. the items themselves - are marked as bought. The status passed in is set on the *order*.